jwt/jwt.html

```diff
@@ -0,0 +1,363 @@
+<html>
+<head>
+  <title>JWT Generator</title>
```

Comment:
There is a newer version of the JWT tool somewhere (I thought it was on the static site already) that has a dropdown for v1 or v2. This is because the aud assertion needs to have the correct version in the url. We'll need that before we can launch v2 to sandbox.